src/exercises/exerciseTypes/audioExerciseOne/BotInput.js

```diff
@@ -0,0 +1,92 @@
+import { useState } from "react";
+import removeAccents from "remove-accents";
```

Comment:
There seems to be A LOT of duplication between this and BottomInput. 
Why do we need this? 
Let's find a way of reusing BottomInput instead of doing all the duplication?